components/nexus-core/src/main/java/org/sonatype/nexus/internal/web/ErrorPageServlet.java

```diff
@@ -98,19 +98,23 @@ public ErrorPageServlet(final Provider<SystemStatus> systemStatus,
     this.webUtils = checkNotNull(webUtils);
   }
 
+  @SuppressWarnings("deprecation")
   @Override
   protected void service(final HttpServletRequest request, final HttpServletResponse response)
       throws ServletException, IOException
   {
     webUtils.addNoCacheResponseHeaders(response);
 
+    @SuppressWarnings("unused")
     String servletName = (String) request.getAttribute(ERROR_SERVLET_NAME);
+    @SuppressWarnings("unused")
     String requestUri = (String) request.getAttribute(ERROR_REQUEST_URI);
     Integer errorCode = (Integer) request.getAttribute(ERROR_STATUS_CODE);
     String errorMessage = (String) request.getAttribute(ERROR_MESSAGE);
-    Class causeType = (Class) request.getAttribute(ERROR_EXCEPTION_TYPE);
+    @SuppressWarnings("unused")
+    Class<?> causeType = (Class<?>) request.getAttribute(ERROR_EXCEPTION_TYPE);
+    @SuppressWarnings("unused")
     Throwable cause = (Throwable) request.getAttribute(ERROR_EXCEPTION);
```

Comment:
you can probably just comment these out for now, we may need to revisit using these details for future refinement, but they aren't presently used, so commenting them out vs. "unused" warning suppression here may be better in the short-term.